src/components/Controls.vue

```diff
@@ -79,11 +157,13 @@
 <script>
 import { mapState, mapGetters } from 'vuex'
 import { Actions, ActionButton } from '@nextcloud/vue'
+import { Popover } from '@nextcloud/vue/dist/Components/Popover'
```

Comment:
Since we use a lot of the components, please use the global import. That way we save quite some bytes in the bundle size.
```suggestion
import { Popover } from '@nextcloud/vue'
```